hazelcast/src/main/java/com/hazelcast/internal/management/operation/ScriptExecutorOperation.java

```diff
@@ -80,14 +74,7 @@ public Object getResponse() {
     protected void writeInternal(ObjectDataOutput out) throws IOException {
         out.writeUTF(engineName);
         out.writeUTF(script);
-        if (bindings != null) {
-            out.writeInt(bindings.size());
-            Set<Map.Entry<String, Object>> entries = bindings.entrySet();
-            for (Map.Entry<String, Object> entry : entries) {
-                out.writeUTF(entry.getKey());
-                out.writeObject(entry.getValue());
-            }
-        } else {
+        if (out.getVersion().isLessThan(V3_10)) {
```

Comment:
Does this need to be `isUnknownOrLessThan()`, since the `Versioned` is new in 3.10? @vbekiaris

Reply:
See other comment. Vassilis, please correct me if I'm wrong.